src/tools/markdown/markdown.ts

```diff
@@ -58,6 +58,13 @@ export class markdown {
         .replace("/blob/", "/");
       return url;
     });
+
+    // Shorten commits links
+    input = input.replace(/(https:\/\/github\.com\/\S*\/commit\/[0-9a-f]{40})/g, function (x) {
+      let hash = x.substr(x.length - 40, 7);
```

Comment:
```suggestion
      const hash = x.substr(0, 7);
```

Reply:
This doesn't work as expected

![image](https://user-images.githubusercontent.com/23432278/120934602-41eddd80-c6ff-11eb-879b-a99c4a8e436b.png)